.pre-commit-config.yaml.jinja

```diff
@@ -101,16 +101,16 @@ repos:
           - --exit-zero
         verbose: true
         additional_dependencies:
-          - isort==4.3.4
-          - pylint-odoo==3.5.0
+          - isort==4.3.21
```

Comment:
We're using isort 5.5.1 in another hook above. Did you try to use that same version here?

Reply:
Yes, but `pylint` requires `isort<5 and >=4.2.5`.